src/components/facebook-login/tests/Facebook.test.js

```diff
@@ -0,0 +1,45 @@
+import {shallow} from 'enzyme';
+import React from 'react';
+import {Provider} from 'react-redux';
+import {MemoryRouter} from 'react-router-dom';
+import configureStore from 'redux-mock-store';
+import Facebook from '../Facebook';
+
+const mockStore = configureStore();
+
+describe('Social Auth Buttons', () => {
+
+	let wrapper;
+	let props;
+
+	beforeEach(() => {
+		const store = mockStore({});
+
+		props = {
+			onSocialSuccess: jest.fn(),
+			styles: {}
+		};
+		wrapper = shallow(
+			<MemoryRouter initialEntries={['/']}>
+				<Provider store={store}>
+					<Facebook {...props} />
+				</Provider>
+			</MemoryRouter>);
+	});
+
+	it('should render facebook button', () => {
+		expect(wrapper).toHaveLength(1);
+		expect(wrapper.find(Facebook)).toHaveLength(1);
+	});
+
+	it('should call facebook return', async() => {
+		// console.log(wrapper.find(FacebookLogin).props());
+		wrapper.find(Facebook);
+
+		// facebook.loginWithFacebook(fBResponse);
```

Comment:
@reiosantos  please remove this commented code

Reply:
This code has been removed in the latest commit.